aquifers/AQUFLUX-02.DATA

```diff
@@ -0,0 +1,307 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2023 Equinor
+-- Copyright (C) 2023 SINTEF ICT, Applied Mathematics
```

Comment:
Are you really sure that you want to list SINTEF ICT here?

Reply:
The case is based on `AQUFET-01.DATA`, which has a copyright claim from 2015. I will update it.